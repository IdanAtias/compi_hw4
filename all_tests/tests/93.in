C x = 5 C;
switch(x)
{
	case 6 C:
		C d = 6 F;
		break;
}
