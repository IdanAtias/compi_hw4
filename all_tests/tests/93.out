-- end scope --
var:d:1:CELSIUS
-- end scope --
var:x:0:CELSIUS
